vowpalwabbit/vw.sln

```diff
@@ -47,30 +41,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_testhelp", "..\cs\testhe
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "vw_common", "..\cs\common\vw_common.csproj", "{E621E022-C1F8-433F-905A-AB9A3DE072B7}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_leaktest", "..\cs\leaktest\cs_leaktest.csproj", "{492EA3A7-8A41-459E-BBC3-5A3FE9DDFC68}"

```

Comment:
Any reason to keep the files for this project?

Reply:
No, good catch